src/libs/datatableproviders/EntityServiceDataProvider.ts

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line lodash-fp/use-fp
+import { ValueKeyIteratee } from 'lodash'
+import _ from 'lodash/fp'
+import { Ajax } from 'src/libs/ajax'
+import { DataProvider, DataProviderFeatures, DeleteTable, EntityQuerySortDirection, GetMetadata, GetPage } from 'src/libs/datatableproviders/DataProvider'
+
+
+export class EntityServiceDataProvider implements DataProvider {
+  features: DataProviderFeatures = {
+    enableTsvDownload: true,
+    enableTypeDeletion: true,
+    enableTypeRenaming: true,
+    enableExport: true,
+    enablePointCorrection: true,
+    enableFiltering: true
+  }
+
+  getPage: GetPage = async (signal: any, _workspaceId: string, entityType: string, pageNumber: number,
```

Comment:
Would it make sense to instantiate the DataProvider with a workspace so that the workspace ID or namespace/name doesn't have to be passed to each method?

```js
const dataProvider = new EntityServiceDataProvider(workspace)
```